package.json

```diff
@@ -1,23 +1,24 @@
 {
   "name": "@expressen/tallahassee",
-  "version": "8.1.0",
+  "version": "9.1.0",
```

Comment:
Should this be version 9.0.0?

Reply:
Yep, breaking. But to keep up with master it is now on 9.1

Reply:
README says `9.0.0` however. Still correct?